app/src/main/kotlin/com/wire/android/ui/calling/OngoingCallViewModel.kt

```diff
@@ -37,13 +42,35 @@ class OngoingCallViewModel @Inject constructor(
 
     init {
         viewModelScope.launch {
-            observeOngoingCall()
+            launch { initializeScreenState() }
+            launch { observeOngoingCall() }
         }
     }
 
+    private suspend fun initializeScreenState() {
+        conversationDetails(conversationId = conversationId)
+            .collect {
+                callEstablishedState = when (it) {
+                    is ConversationDetails.Group -> {
+                        callEstablishedState.copy(
+                            conversationName = it.conversation.name
+                        )
+                    }
+                    is ConversationDetails.OneOne -> {
+                        callEstablishedState.copy(
+                            conversationName = it.otherUser.name,
+                            avatarAssetId = UserAvatarAsset(it.otherUser.completePicture ?: "")
```

Comment:
Wouldn't be easier to handle the user avatar later if we check if it is just null or not instead of checking if it is null, empty or something else?

Reply:
```suggestion
                            avatarAssetId = conversationDetails.otherUser.completePicture?.let { UserAvatarAsset(it) }
```

Reply:
Something like:
```suggestion
                    avatarAssetId = conversationDetails.otherUser.completePicture?.let { UserAvatarAsset(it) }
```

So it will only add `UserAvatarAsset` if it is not null, if it is null, then `avatarAssetId == null` and it will be easier to handle later on, what do you think?